docs/guide/yaml/bigger-blueprint.md

```diff
@@ -0,0 +1,43 @@
+---
+title: A Bigger Blueprint
```

Comment:
i don't think there's enough content in this page to justify it being its own.  combine with the following "Multiple Services and DI" page?  (not sure but i think i'd keep the MS+DI title for the merged page as it is more informative in the menu)

(there's also the issue that the app deployed at the end of this page won't work as there is no DB, but it won't be clear reading this page!)